src/main/java/competition/subsystems/drive/commands/SwerveDriveWithJoysticksCommand.java

```diff
@@ -270,7 +270,11 @@ public void execute() {
         if (drive.isCollectorRotationActive()) {
             centerOfRotationInches = new XYPair(arms.getCurrentXZCoordinates().x, 0);
         }
-        
+
+        if (drive.isGamePieceRotationActive()) {
+            centerOfRotationInches = new XYPair(20, 0);
```

Comment:
Note that 0,0 would be in the dead center of the robot - so this is just 20 inches away from that, which in turn is about ~2-3" in front of the bumper. You probably want a larger number here.